File: mullins-roger/shortest-in-string.js
function findShort(s) {
  let words = s.split(" ");
  console.log(words);
  let shortestWord = words[0];
  for (let word of words) {
    console.log("Word is ", word);
    console.log("shortestWord is ", shortestWord);
    if(word.length<=shortestWord.length) {
      console.log(word, " is shorter than ", shortestWord);
      shortestWord=word;
      console.log(shortestWord);
    };
  }
  return shortestWord.length;
}

File: mullins-roger/shortest-in-string.py
def find_short(s):
    # your code here
    in_array = s.split()
    l = in_array[0]
    for word in in_array:
        print(word, len(word))
        if len(word) < len(l):
            l = word
    return len(l) # l: shortest word length
